tests/consensus_tests/test_resharding_down_abort_converges_after_crash.py

```
@@ -0,0 +1,115 @@
+    # Restart (liveness only — the stuck shard may never pass /readyz). The victim
+    # replays the abort against an already-cleared, persisted resharding_state and
+    # converges.
+    peer_uris[victim_idx] = start_peer(
+        peer_dirs[victim_idx], f"peer_restart_{victim_idx}.log", alive_uri, port=victim_port,
+    )
+    wait_for_peer_online(peer_uris[victim_idx], path="/healthz", wait_for_timeout=60)
+
+    try:
+        for uri in peer_uris:
+            wait_for_collection_resharding_operations_count(uri, COLLECTION, 0, wait_for_timeout=30)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)